drivers/hmis/lib/form_data/tarrant_county/assessments/caseworker_exit.json

```
@@ -0,0 +1,611 @@
+            "custom_field_key": "caseworker_exit_beginning_gas_score"
+          }
+        },
+        {
+          "text": "Beginning GAS Score Date",
+          "type": "DATE",
+          "link_id": "caseworker_exit_beginning_gas_score_date",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_beginning_gas_score_date"
+          }
+        },
+        {
+          "text": "Exit GAS Score",
+          "helper_text": "Target 40.00",
+          "type": "INTEGER",
+          "link_id": "caseworker_exit_end_gas_score",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_end_gas_score"
+          }
+        },
+        {
+          "text": "Exit GAS Score Date",
+          "type": "DATE",
+          "link_id": "caseworker_exit_end_gas_score_date",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_end_gas_score_date"
+          }
+        },
+        {
+          "text": "Difference/Change in GAS score",
+          "type": "INTEGER",
+          "link_id": "caseworker_exit_gas_score_change",
+          "read_only": true,
+          "autofill_values": [
+            {
+              "formula": "caseworker_exit_end_gas_score - caseworker_exit_beginning_gas_score",
+              "autofill_when": [
+                {
+                  "operator": "EXISTS",
+                  "question": "caseworker_exit_end_gas_score",
+                  "answer_boolean": true
+                },
+                {
+                  "operator": "EXISTS",
+                  "question": "caseworker_exit_beginning_gas_score",
+                  "answer_boolean": true
+                }
+              ],
+              "autofill_behavior": "ALL",
+              "autofill_readonly": true
+            }
+          ],
+          "mapping": {
+            "custom_field_key": "caseworker_exit_gas_score_change"
+          }
+        },
+        {
+          "text": "Percent of GAS Change",
+          "type": "INTEGER",
+          "link_id": "caseworker_exit_gas_score_percent_change",
+          "read_only": true,
+          "autofill_values": [
+            {
+              "formula": "(caseworker_exit_end_gas_score - caseworker_exit_beginning_gas_score) / 40 * 100",
+              "autofill_when": [
+                {
+                  "operator": "EXISTS",
+                  "question": "caseworker_exit_end_gas_score",
+                  "answer_boolean": true
+                },
+                {
+                  "operator": "EXISTS",
+                  "question": "caseworker_exit_beginning_gas_score",
+                  "answer_boolean": true
+                }
+              ],
+              "autofill_behavior": "ALL",
+              "autofill_readonly": true
+            }
+          ],
+          "mapping": {
+            "custom_field_key": "caseworker_exit_gas_score_percent_change"
+          }
+        },
+        {
+          "text": "Where will the client be residing? (Please check one).",
```

This feels more appropriate since it's a single-select dropdown
```suggestion
          "text": "Where will the client be residing?",
```